### PR TITLE
new lesson creating gradle project

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,132 @@
...
-referencing.
+IntelliJ creates the new project with the sample `Main` class.
+   
+![new project view](https://curriculum-content.s3.amazonaws.com/6002/creating-a-gradle-project/projectfolders.png)
+
+Press the green run button to execute the `main` method.
+The `.gradle` folder is created after the code compiles and runs.
+
+
+## Maven Directory Structure
```
Comment (kcleland0818) [inline README.md]: Typo: Gradle Directory Structure??

File: README.md [hunk 2]
```diff
@@ -2,31 +2,132 @@
...
+		│   │   ├── java
+		│   │   └── resources
+		│   └── test
+		│       ├── java
+		│       └── resources
+		├── build.gradle
+		└── settings.gradle	
+```
+
+The structure of the `src` folder is similar to Maven's. Gradle does not use a `pom.xml` file
```
Comment (kcleland0818) [inline README.md]: Missing period after file.

"Gradle does not use a `pom.xml` file"

**Review by kcleland0818 — CHANGES_REQUESTED** [review]
Small typos

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: fixed.

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: fixed.

**Review by kcleland0818 — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**